client/shaders/second_stage/opengl_fragment.glsl

```diff
@@ -61,14 +61,17 @@ vec3 uncharted2Tonemap(vec3 x)
 
 vec4 applyToneMapping(vec4 color)
 {
-	const float exposureBias = 2.0;
+	color = vec4(pow(color.rgb, vec3(2.2)), color.a);
+	const float gamma = 1.6;
```

Comment:
These constants come out of nowhere somehow.
In any case I looked up the original commit (9df79a4b2d68979c3e15797d518d957787ba4e21) that changed this behavior, and these constants were there before.
So that's good then as long as we're happy with the visual outcome - which I think we are.